config/crd/bases/batch.volcano.sh_jobs.yaml

```diff
@@ -19,38 +19,7 @@ spec:
     singular: job
   scope: Namespaced
   versions:
-  - additionalPrinterColumns:
```

Comment:
These changes is commited to volcano.sh/volcano/pkg/api/ after you move api to volcano/api/.  Need to update the chagnes to volcano.sh/api master first and then refresh this pr.